examples/js/controls/OrbitControls.js

```diff
@@ -254,11 +244,9 @@ THREE.OrbitControls = function ( object, domElement ) {
 	var EPS = 0.000001;
 
 	// current position in spherical coordinates
-	var theta;
-	var phi;
+	var spherical = new THREE.Spherical();
+	var sphericalDelta = new THREE.Spherical();
```

Comment:
I left the sphericalDelta class in, because it is explanatory, but I do not use the add/multipleScalar methods anymore, as I do see those as potentially misleading.